README.md

```diff
@@ -170,20 +180,87 @@ platform, but it's also been tested on the OrangeCrab and ButterStick.
 
    For the Arty A7 A35, set `FLASH_ADDRESS` to `0x300000` and pass `-f a35`.
    ```
-   microwatt/openocd/flash-arty -f a100 build/microwatt_0/arty_a7-100-vivado/microwatt_0.bit
-   microwatt/openocd/flash-arty -f a100 dtbImage.microwatt.elf -t bin -a $FLASH_ADDRESS
+   sudo microwatt/openocd/flash-arty -f a100 build/microwatt_0/arty_a7-100-vivado/microwatt_0.bit
+   sudo microwatt/openocd/flash-arty -f a100 dtbImage.microwatt.elf -t bin -a $FLASH_ADDRESS
    ```
 
 5. Connect to the second USB TTY device exposed by the FPGA
 
    ```
-   minicom -D /dev/ttyUSB1
+   sudo minicom -D /dev/ttyUSB1
    ```
+   If the cable was plugged in, unplug it when you're done and you'll see the serial console on the minicomputer. If you can't see the serial console, see below.
+   
+   go to Serial Port Setup; last two lines are Hardware and Software Flow control; just set NO both)
 
+   - https://stackoverflow.com/questions/3913246/cannot-send-character-with-minicom
+   ```
+   sudo minicom -s; 
+   ```
+   
    The gateware has firmware that will look at `FLASH_ADDRESS` and attempt to
    parse an ELF there, loading it to the address specified in the ELF header
    and jumping to it.
 
+6. SSH login
+
+  Check  your DHCP server
+
+  ```
+  $ cat /etc/default/isc-dhcp-server
+  # Defaults for isc-dhcp-server (sourced by /etc/init.d/isc-dhcp-server)
+  # Path to dhcpd's config file (default: /etc/dhcp/dhcpd.conf).
+  #DHCPDv4_CONF=/etc/dhcp/dhcpd.conf
+  #DHCPDv6_CONF=/etc/dhcp/dhcpd6.conf
+  # Path to dhcpd's PID file (default: /var/run/dhcpd.pid).
+  #DHCPDv4_PID=/var/run/dhcpd.pid
+  #DHCPDv6_PID=/var/run/dhcpd6.pid
+  # Additional options to start dhcpd with.
+  #	Don't use options -cf or -pf here; use DHCPD_CONF/ DHCPD_PID instead
+  #OPTIONS=""
+  # On what interfaces should the DHCP server (dhcpd) serve DHCP requests?
+  #	Separate multiple interfaces with spaces, e.g. "eth0 eth1".
+  INTERFACESv4="enp6s0"
+  INTERFACESv6="enp6s0"
+  ```
+
+  ```
+  $ sudo ifconfig enp6s0 192.168.0.1
+  $  sudo /etc/init.d/isc-dhcp-server restart
```

Comment:
Hmmm, it seems dangerous to me to tell people to set up a DHCP server without at least explaining when this might be necessary and what potential problems it might cause. I would rather just tell people to query the DHCP server on their network to find out what address was assigned, or if that is not possible, log in to the microwatt system on the console and run `ifconfig` (or `ip addr list`) to find out.

It should be possible for people to build and run microwatt without ever using sudo.